src/test/jtx/impl/AMM.cpp

```diff
@@ -745,7 +745,7 @@ AMM::expectAuctionSlot(auto&& cb) const
             static_cast<STObject const&>(amm->peekAtField(sfAuctionSlot));
         if (auctionSlot.isFieldPresent(sfAccount))
         {
-            auto const slotFee = auctionSlot.getFieldU32(sfDiscountedFee);
+            auto const slotFee = auctionSlot[~sfDiscountedFee].value_or(0);
```

Comment:
This can safely be changed back to the original code `auto const slotFee = auctionSlot.getFieldU32(sfDiscountedFee);` or `auto const slotFee = auctionSlot[sfDiscountedFee];`. If the field isn't set, it'll return 0.